openedx/core/djangoapps/user_api/accounts/serializers.py

```diff
@@ -88,6 +88,7 @@ def to_representation(self, user):
             "level_of_education": AccountLegacyProfileSerializer.convert_empty_to_None(profile.level_of_education),
             "mailing_address": profile.mailing_address,
             "requires_parental_consent": profile.requires_parental_consent(),
+            "account_privacy": UserPreference.get_value(user, 'account_privacy'),
```

Comment:
In order to pave the way for reducing the number of queries for the teams API when they implement prefetching, I'd like to see this rearranged to pull user.preferences.all(), and pull the privacy preference from that.  It's a slightly larger response from the database, as it's pulling all the preferences, rather than just the account_privacy, but it facilitates reducing a linear number of queries to a single query when used in the teams API.  Example change: https://gist.github.com/jcdyer/7d9a99d8e989898cadd5